.evergreen.yml

```diff
@@ -11478,6 +11478,18 @@ buildvariants:
       mongosh_test_e2e_force_fips: ""
     tasks:
       - name: e2e_tests_linux_ppc64le
+  - name: e2e_tests_rhel9_power_small
+    display_name: "RHEL 9 PPC (E2E tests)"
+    run_on: rhel9-power-small
+    tags: []
+    expansions:
+      executable_os_id: "linux-ppc64le"
+      disable_openssl_shared_config_for_bundled_openssl: false
```

Comment:
Ideally, as part of this patch we'd also rip out all mentions of `disable_openssl_shared_config_for_bundled_openssl` and `disableOpenSslSharedConfig` entirely, their presence was always a workaround for this issue